content/en/docs/concepts/security/pod-security-policy.md

```diff
@@ -332,7 +335,15 @@ The output is similar to this
 pod "pause" created
 ```
 
-It works as expected! But any attempts to create a privileged pod should still
+It works as expected! You can verify that the pod was validated against the
+newly created PodSecurityPolicy:
+
+```shell
+kubectl-user get pod pause -o yaml | grep kubernetes.io/psp
+kubernetes.io/psp: example
```

Comment:
Please [separate the command from its output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output).

Reply:
I did it also for the rest, I am not sure how much "the output is similar to this" I should imply instead of writing but this seems ok!

Reply:
https://github.com/kubernetes/website/pull/34675/commits/8a4e62fb766e8efc296ca91823b210941fb24bcd